x-pack/plugins/alerting/server/routes/clone_rule.ts

```diff
@@ -69,11 +69,12 @@ const rewriteBodyRes: RewriteResponseCase<PartialRule<RuleTypeParams>> = ({
     : {}),
   ...(actions
     ? {
-        actions: actions.map(({ group, id, actionTypeId, params }) => ({
+        actions: actions.map(({ group, id, actionTypeId, params, lastTriggerDate }) => ({
           group,
           id,
           params,
           connector_type_id: actionTypeId,
+          ...(lastTriggerDate !== undefined && { last_trigger_date: lastTriggerDate }),
```

Comment:
I think we should also add the new `frequency` fields (notifyWhen, throttle, summary) here...